samples/managed-identity/readme.md

```diff
@@ -20,10 +20,28 @@
 If you want to test with pod identity as well, please follow this link to setup pod identity on your cluster:
 https://azure.github.io/aad-pod-identity/docs/demo/standard_walkthrough/
 
-1. Deploy the pod identity components from Step 1 in the link.
+1. Deploy the pod identity components from Step 1 in the link. 
+ex: this deploys CRD's to the cluster
+```
+kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/deployment-rbac.yaml
+
+# For AKS clusters, deploy the MIC and AKS add-on exception by running -
+kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/mic-exception.yaml 
+
+```
 2. Assign the appropriate permissions to the AKS Managed Identity: https://azure.github.io/aad-pod-identity/docs/getting-started/role-assignment/
+ex: Run the roleassigment bash script
+```
+./samples/managed-identity/roleAssignment.sh
+```
 3. Note: If you are using an AKS cluster with version > 1.17 (Kubenet enabled by default) - an additional step is required https://azure.github.io/aad-pod-identity/docs/configure/aad_pod_identity_on_kubenet/
```

Comment:
minimum version for supported aks clusters is 1.25, probably should just make this a regular step

Reply:
updated, removed this step which was not required or it didn't give me any error regarding kubenet while testing via bridge.